django_ai_assistant/models.py

```diff
@@ -47,10 +47,9 @@ class Message(models.Model):
     A message can be sent by a user or the AI assistant.\n
     The message data is stored as a JSON field called `message`."""
 
-    id: int  # noqa: A003
     thread = models.ForeignKey(Thread, on_delete=models.CASCADE, related_name="messages")
     """Thread to which the message belongs."""
-    thread_id: int  # noqa: A003
+    thread_id: Any  # noqa: A003
```

Comment:
Please see if this is problematic @fjsj

Reply:
LGTM but might cause editor linter errors. We can see that later.